examples/charts/horizon.py

```diff
@@ -25,7 +25,8 @@
 output_file("horizon.html")
 
 hp = Horizon(
-    xyvalues, index='Date',
+    xyvalues, x_names=['Date'],
```

Comment:
I agree but I think we should be consistent. Builder already had `x_range` and `y_range` there so I tried to follow that pattern (which is also used in the `plotting` interface).